plugins/rapture/nexus-coreui-plugin/pom.xml

```diff
@@ -67,6 +67,24 @@
       <artifactId>goodies-testsupport</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.spockframework</groupId>
+      <artifactId>spock-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.spockframework</groupId>
+      <artifactId>spock-guice</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.sonatype.nexus</groupId>
+      <artifactId>nexus-extender</artifactId>
```

Comment:
Docs have gotten much better with the 1.0 release: http://spockframework.github.io/spock/docs/1.0/
Couple of concrete notes:
- the when/then and other blocks are all Java labels, allowing the test runner to pre-process them before run; this can be a bit confusing because the code is not necessarily executed top-to-bottom
- where blocks essentially turn the method into the equivalent of a JUnit Parameterized test